include/OpenSpaceToolkit/Astrodynamics/Trajectory/CoordinatesSubsets/Invariant.hpp

```diff
@@ -0,0 +1,61 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Trajectory_CoordinatesSubsets_Invariant__
+#define __OpenSpaceToolkit_Astrodynamics_Trajectory_CoordinatesSubsets_Invariant__
+
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/CoordinatesBroker.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace trajectory
+{
+namespace coordinatessubsets
```

Comment:
```suggestion
namespace coordinatessubset
```
To match the naming of the class. What do think?